renderer/components/ConfigBox.tsx

```diff
@@ -191,30 +187,18 @@ class ConfigBoxComponent extends React.Component {
   }
 }
 
-ConfigBoxComponent.propTypes = {
-  shouldShow: PropTypes.bool.isRequired,
-  hide: PropTypes.func.isRequired,
-  ipAddress: PropTypes.string.isRequired,
-  stationNumber: PropTypes.number.isRequired,
-  onIPChange: PropTypes.func.isRequired,
-  fcAddress: PropTypes.string.isRequired,
-  onFCUpdate: PropTypes.func.isRequired,
-};
-
-const mapDispatchToProps = dispatch => ({
-  onIPChange: (ipAddress) => {
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  onIPChange: (ipAddress: string) => {
     dispatch(ipChange(ipAddress));
   },
-  onFCUpdate: (config) => {
+  onFCUpdate: (config: Config) => {
     dispatch(updateFieldControl(config));
   },
 });
 
-const mapStateToProps = state => ({
-  stationNumber: parseInt(state.fieldStore.stationNumber, 10),
+const mapStateToProps = (state: ApplicationState) => ({
+  stationNumber: state.fieldStore.stationNumber,
```

Comment:
The `ApplicationState` type is declared globally so no need to import from the top of the file. This type is exported from `renderer/types/types.d.ts`. The reason I declared this globally was because was so it can be widely accessible across the components. Ideally, we want to limit the scope of types so that we know where they are defined. This would make it easier to debug but `ApplicationState` is an exception.